docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+services:
+  aws-quick-ui:
+    container_name: "aws-quick-ui"
+    build: 
+      context: .
+    ports:
+      - 3000:3000
+    environment:
+      - LOCALSTACK_ENDPOINT=http://localstack:4566
+      - LOCALSTACK_REGION=us-west-2
+      - AWS_ACCESS_KEY_ID=localstack
+      - AWS_SECRET_ACCESS_KEY=localstack
+
+    networks:
+      - localnetwork
+    depends_on:
+      - localstack
+
+  localstack:
+    container_name: "${LOCALSTACK_DOCKER_NAME:-localstack-main}"
+    image: localstack/localstack
+    ports:
+      - "127.0.0.1:4566:4566"            # LocalStack Gateway
+      - "127.0.0.1:4510-4559:4510-4559"  # external services port range
+    environment:
+      # LocalStack configuration: https://docs.localstack.cloud/references/configuration/
+      - DEBUG=${LOCALSTACK_DEBUG:-0}
+      - SERVICES=${LOCALSTACK_SERVICES}
+    networks:
+      - localnetwork
+    volumes:
+      - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"
+      - "/var/run/docker.sock:/var/run/docker.sock"
+
+networks:
+  localnetwork:
```

Comment:
not sure if we have to spin localstack along with quick ui
I believe if someone uses this app, they already do have localstack installed

Reply:
yes you are right. I can make a separate docker-compose file to spin up localstack container. If some one want to spin localstack container they can do easily